src/codegen/llvm.zig

```diff
@@ -705,7 +701,7 @@ const DataLayoutBuilder = struct {
                 force_pref = true;
             },
             .float => switch (self.target.cpu.arch) {
-                .aarch64_32, .amdgcn => if (size == 128) {
```

Comment:
Likewise, I could not convince myself that this made any difference since AArch64 (ILP32 or not) just has a normal binary128 `long double`/`f128`.